### ISSUE TITLE
Meta information in landing pages is not inherited

### ISSUE THREAD

**Issue by michnhokn** [shopware/platform#3712]
### PHP Version

8.2

### Shopware Version

trunk

### Expected behaviour

The meta information of the landing page should use the translated version of the field.

### Actual behaviour

If the language has no defined name or meta title, it will not fall back to the default languages, the meta title will be empty:
https://github.com/shopware/shopware/blob/9db9338117f00139a366a07672da9b768692f6b7/src/Storefront/Page/LandingPage/LandingPageLoader.php#L52-L57

### How to reproduce

Simply create two languages and add a landing page. The meta title in the default language will work.
If you do not translate the title in the other language, it will NOT fall back based on the language id chain and will therefore be empty.

**Comment by mitelg** [comment]
https://issues.shopware.com/issues/NEXT-31206 seems to be the same issue. could we close the issue here then?

**Comment by michnhokn** [comment]
@mitelg Yeah sure. Tbh I was not looking there ;)

**Comment by Songworks** [comment]
@mitelg @michnhokn  Sorry to necro an old issue but we came upon this exact issue recently in v6.6.

Has [NEXT-31206](https://issues.shopware.com/issues/NEXT-31206) been fixed or resolved since you guys closed this issue?
The issue tracker url is not reachable anymore and latest code here on Github still contains the reported issue.


If anybody else comes across this, here's a git diff to fix it:
```diff
diff --git a/Page/LandingPage/LandingPageLoader.php b/Page/LandingPage/LandingPageLoader.php
index 404db0e..78ddc18 100644
--- a/Page/LandingPage/LandingPageLoader.php
+++ b/Page/LandingPage/LandingPageLoader.php
@@ -49,11 +49,15 @@ class LandingPageLoader
 
         $page->setLandingPage($landingPage);
 
+        $metaTitle       = $landingPage->getTranslation('metaTitle') ?? $landingPage->getTranslation('name');
+        $metaDescription = $landingPage->getTranslation('metaDescription');
+        $metaKeywords    = $landingPage->getTranslation('keywords');
+
         $metaInformation = new MetaInformation();
-        $metaTitle = $landingPage->getMetaTitle() ?? $landingPage->getName();
-        $metaInformation->setMetaTitle($metaTitle ?? '');
-        $metaInformation->setMetaDescription($landingPage->getMetaDescription() ?? '');
-        $metaInformation->setMetaKeywords($landingPage->getKeywords() ?? '');
+
+        $metaInformation->setMetaTitle((string) $metaTitle);
+        $metaInformation->setMetaDescription((string) $metaDescription);
+        $metaInformation->setMetaKeywords((string) $metaKeywords);
         $page->setMetaInformation($metaInformation);
 
         $this->eventDispatcher->dispatch(

```

Basically the same way `\Shopware\Storefront\Page\Navigation\NavigationPageLoader::loadMetaData` does it.

Cheers!

[NEXT-31206]: https://shopware.atlassian.net/browse/NEXT-31206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by mitelg** [comment]
hey @Songworks 

thanks for bringing that up again. the mentioned Jira ticket was closed as "won't do". But there was already a PR, which was not finished unfortunately: https://github.com/shopware/shopware/pull/3757

Would be nice, if you could put your fix into a PR. it looks promising 👍 

As the Jira ticket is not reachable anymore, I will re-open this one again

**Comment by Songworks** [comment]
@mitelg Thanks for the quick response.
I'll have a look at creating a PR for it soon-ish.

Cheers!

**Comment by keulinho** [comment]
already fixed here: https://github.com/shopware/shopware/pull/8137

the fix was released with 6.7.1.0
